install/first_google_boot.sh

```diff
@@ -207,6 +207,10 @@ function extract_spinnaker_kube_credentials() {
   local kube_project=$(get_instance_metadata_attribute "kube_project")
   local kube_zone=$(get_instance_metadata_attribute "kube_zone")
 
+  clear_instance_metadata "kube_cluster"
```

Comment:
I like to clear them at the end so that if you encounter a failure you can fix it then run first_google_boot again and it will automatically pick up where it left off. If you clear this first and hit an error downloading gcloud below (for example) then there is no recovery path because you've destroyed the evidence. If you want until later then next time through it will see the attributes and attempt [again] to process them.